### ISSUE TITLE
Install script for bash has DOS line endings

### ISSUE THREAD

**Issue by teidesu** [nimsandu/spicetify-bloom#32]
Basically the title. Because of that shell doesn't correctly process it and it doesn't work:

```bash
$ curl -fsSL https://raw.githubusercontent.com/nimsandu/spicetify-bloom/main/install.sh | sh

: command not found
: invalid option -
set: usage: set [--abefhkmnptuvxBCHP] [-o option] [arg ...]

$ ./install.sh
zsh: ./install.sh: bad interpreter: /bin/sh^M: no such file or directory
```

Side note: the script doesn't work at all on macOS, there are numerous errors with it

**Comment by nimsandu** [comment]
can you kindly give me a hand to fix that
I actually don't know about shell scripts  
@teidesu please 🥲

**Comment by nimsandu** [comment]
please @teidesu 🥺

**Comment by q60** [comment]
fixed in #40

**Comment by nimsandu** [comment]
@q60 thanx
Im going to close the issue
@teidesu don't hesitate to open another issue if you find another issue

Thanks everyone
